schedules/toggle_schedule.py

```diff
@@ -0,0 +1,20 @@
+import asyncio
+
+from temporalio.client import Client
+
+
+async def main():
+    client = await Client.connect("localhost:7233")
+    handle = client.get_schedule_handle(
+        "workflow-schedule-id",
+    )
+
+    await handle.pause(note="Pausing the schedule for now")
+
+    desc = await handle.describe()
+
+    print(f"Describe the schedule's state: {desc.schedule.state}")
```

Comment:
"Describe the schedule's state:" is more like a command asking someone to do something rather than a description of what you're dumping.

Also, is it worth showing the entire state on these samples?

Reply:
This print wording still exists on `trigger_schedule.py`